src/SFA.DAS.ApprenticeFeedback.Api/Controllers/FeedbackTransactionController.cs

```diff
@@ -73,5 +73,23 @@ public async Task<IActionResult> ProcessEmailTransaction(int feedbackTransaction
             }
 
         }
+
+        [HttpPost("{feedbackTransactionId}/track-click")]
+        public async Task<IActionResult> TrackEmailTransactionClick(long feedbackTransactionId, [FromBody] TrackEmailTransactionClickCommand command)
+        {
+            try
+            {
+                var result = await _mediator.Send(command);
+                if (null == result) return NotFound();
```

Comment:
The handler for this command doesn't seem to return null?

Reply:
Removed